modules/configuration/src/main/java/org/apache/ignite/internal/configuration/asm/ConfigurationAsmGenerator.java

```diff
@@ -3406,4 +3425,21 @@ private static void addInternalSchemaTypesMethod(
                 .append(getThisFieldCode(mtd, internalSchemaTypesFieldDef))
                 .retObject();
     }
+
+    /**
+     * Adds an override for the {@link InnerNode#isExtendAbstractConfiguration()} method that returns {@code true}.
```

Comment:
`isExtendAbstractConfiguration` is not a good name from the grammatical perspective. It should either be ` extendsAbstractConfiguration` (which I prefer) or `doesExtendAbstractConfiguration`

Reply:
Renamed to `extendsAbstractConfiguration`